### ISSUE TITLE
Put DC2/HSC/DES data in NERSC for notebook documentation compilation

### ISSUE THREAD

**Issue by m-aguena** [LSSTDESC/CLMM#585]
Should we put the data for the `DC2/HSC/DES` notebooks somewhere in NERSC/CC for easy compilation of documentation for the website?

**Comment by combet** [comment]
Yes, that's be great. It's public HSC and DES data right, so there's no concern about hosting this small subset of data  ourselves?
